### ISSUE TITLE
Performance from the provided weights

### ISSUE THREAD

**Issue by halbielee** [YudeWang/SEAM#10]
Hello! 

Thanks for the sharing the code.

I ran your code with the trained weights and got lower performance than the paper reported. 
60.076 % mIOU for the validation set.

My inference step is
1) Infer the CAM(npy files) from the infer_SEAM.py with the trained model
2) Infer the Segmentation map(png files) from the infer_aff.py with the trained model
3) evaluate the png files with gt files.

Is there anything I missed?
Than you!

**Comment by YudeWang** [comment]
Hi @halbielee ,
The generated pseudo segmentation labels are evaluated on PASCAL VOC 2012 train set (pixel-level gt of train set are not available during training).
The segmentation model is retrained on these pseudo segmentation labels then evaluated on both validation and test set.

**Comment by halbielee** [comment]
Oh! 
You are right. 

I was confused that the result of Affinitynet was a segmentation result.

Thanks for the reply!
